### PR TITLE
Compatibility with Keycloak 8.0.1

### PR DESCRIPTION
**Pull request by gmatev**
Keycloak 8.0 introduces some breaking changes to the interfaces. This PR adjusts for these. 

Tested manually.

### CODE REVIEW

File: src/main/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProvider.java [hunk 1]
```diff
@@ -68,13 +53,19 @@ public void close() {
     }
 
     @Override
-    public boolean verify(String rawPassword, CredentialModel credential) {
-        return BCrypt.checkpw(rawPassword, credential.getValue());
+    public boolean verify(String rawPassword, PasswordCredentialModel credential) {
+        return BCrypt.checkpw(rawPassword, credential.getPasswordSecretData().getValue());
```
Comment (leroyguillaume) [inline src/main/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProvider.java]: One day, I'll replace this library by https://github.com/patrickfav/bcrypt which is more complete...

**Comment by LordGaav** [comment]
Purely from reading the code, there seem to be no big changes apart from implementing the new interface for PasswordCredentialModel and verify. Not storing the salt separately from the hash is also not new, but mentioned explicitly now as a comment.

Is this conclusion correct? Looking forward to a release.

**Comment by leroyguillaume** [comment]
Merged with #14

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.leroyguillaume</groupId>
     <artifactId>keycloak-bcrypt</artifactId>
-    <version>1.1.0</version>
+    <version>1.2.0</version>
```
[the remark this answers is not in the document]
Comment (gmatev) [reply pom.xml]: Sure. Your call

**Pull request closed without merging**